docs/tutorials/stateful-application/vm-style-workload-using-stateful-set.md

```diff
@@ -0,0 +1,257 @@
+---
+assignees:
+- bprashanth
+- enisoc
+- erictune
+- foxish
+- janetkuo
+- kow3ns
+- smarterclayton
+
+---
+
+{% capture overview %}
+
+This page helps you become familiar with the runtime initialization of [Stateful Sets](/docs/user-guide/petset). 
```

Comment:
This is not necessarily new to this change, but the topic of this item is "VM-style workload" but nothing in the first 40 lines explains why that is.  A sentence in the overview or objectives that explains how the title relates to what is being shown would be good.